specs/dwritecore/DWriteCore.md

```diff
@@ -78,33 +78,272 @@ with interoperability issues.
+a typographic family name or a WWS family name. (It can also be an RBIZ name or full name.) In the
+example above, an application can avoid ambiguity by specifying "Legacy" or "Legacy Soft" as the
+family name. These two cases are handled as follows:
+
+  - "Legacy Soft" does not match a typographic family name, so only fonts in the WWS family
+    are considered. These can be differentiated by weight, so there is no ambiguity.
+
+  - "Legacy" matches both a typographic family and a WWS family. All fonts in the typographic
+    family are considered, but presence in the WWS family is used as a tie-breaker. For example,
+    Legacy-Regular.ttf has the same weight as Legacy-Soft.ttf, but the former is preferred because
+    it is in the "Legacy" WWS family.
+
+Document and application compatibility issues are mitigated by allowing different types of font
+names to be specified and also by providing a method of deriving axis values from weight, stretch,
+and style parameters.
+
+Suppose an application has family name, weight, stretch, style, and font size parameters. For
+example, these might come from a document's backing store. The application can first pass the
+weight, stretch, style, and font size to the `IDWriteFontSet4::GetDerivedFontAxisValues` method
+to map the input parameters to _derived axis values_. If the application also has explicit axis
+values, it should pass these to `GetDerivedAxisValues` as well. The derived axis values will
+not include any axes there were passed in explicitly. This ensures that explicit axes take
+precedence over derived axis values.
+
+Once the application has axis values (derived, explicit, or both), it can pass the family name
+and axis values to the `IDWriteFontSet4::GetMatchingFonts` method. This method returns an
+`IDWriteFontSet4` representing a list of matching fonts in priority order. The specified family
+name can be a typographic family name, WWS family name, RBIZ family name, or full name.
+
+## Hybrid Font Selection APIs
+
+The hybrid font selection model is implemented by the following `IDWriteFontSet4` methods:
+
+  - The `GetDerivedFontAxisValues` method converts font size, weight, stretch, and style parameters
+    to the corresponding axis values. Any explicit axis values passed in by the client are excluded
+    from the derived axis values.
+
+  - The `GetMatchingFonts` method returns a prioritized list of matching fonts given a family name
+    and array of axis values. As described above, the family name parameter can be a typographic
+    family name, WWS family name, RBIZ family name, or full name. (Full name is allowed because
+    GDI allows font matching by full name.)
+
+The following other DirectWrite APIs also use the hybrid font selection algorithm:
+
+  - [`IDWriteFontCollection2::GetMatchingFonts`](https://docs.microsoft.com/en-us/windows/win32/api/dwrite_3/nf-dwrite_3-idwritefontcollection2-getmatchingfonts)
```

Comment:
Nit: Removed language from URL to avoid forcing non-English folks into English docs.

```suggestion
  - [`IDWriteFontCollection2::GetMatchingFonts`](https://docs.microsoft.com/windows/win32/api/dwrite_3/nf-dwrite_3-idwritefontcollection2-getmatchingfonts)
```